themes/theme-gmd/components/Search/components/Suggestions/components/List/index.jsx

```diff
@@ -36,19 +42,34 @@ class SuggestionList extends Component {
     }
 
     return (
```

Comment:
fyi: also applied portals to `themes/theme-ios11/pages/Browse/components/SearchField/components/SuggestionList/index.jsx`